web/modules/custom/dpl_fbi/src/Plugin/Field/FieldWidget/WorkIdSearchForMaterialWidget.php

```diff
@@ -22,11 +22,12 @@ final class WorkIdSearchForMaterialWidget extends WidgetBase {
 
   /**
    * {@inheritdoc}
+   *
+   * @throws \Random\RandomException
    */
   public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state): array {
 
-    $uniqueFieldId = $items->getFieldDefinition()->getUniqueIdentifier();
-    $identifier = "{$uniqueFieldId}-{$delta}";
+    $identifier = bin2hex(random_bytes(16));
```

Comment:
Digging a bit deeper into this an alternate approach would be to use [`Html::getUniqueId()`](https://github.com/drupal/drupal/blob/b12dd10f7558579d1268b16a76f278023fc78e2a/core/lib/Drupal/Component/Utility/Html.php#L149-L183) but I guess that is stuff to try out another day 😅 .